frontend/src/lib/email/EmailVerificationStatus.svelte

```diff
@@ -29,6 +30,9 @@
     }
   }
 
+  $: emailResult = useEmailResult();
+  $: requestedEmail = useRequestedEmail();
```

Comment:
I don't think making these reactive does anything. Since there's no variables referenced by the expression the value would only be updated if the function changed, but that won't happen either.

Reply:
True. I don't think they need to be reactive.